record/record.controller.js

```diff
@@ -90,15 +90,17 @@
         };
 
         vm.sharePopup = function() {
+            var tuple = $rootScope.tuple;
             modalUtils.showModal({
                 templateUrl: UriUtils.chaiseDeploymentPath() + "common/templates/shareCitation.modal.html",
                 controller: "ShareCitationController",
-                windowClass: "share-citation",
+                windowClass: "chaise-share-citation",
                 controllerAs: "ctrl",
                 resolve: {
                     params: {
-                        citation: $rootScope.tuple.citation,
-                        permalink: $window.location.href
+                        citation: tuple.citation,
+                        permalink: $window.location.href,
+                        displayname: $rootScope.reference.table.name+'_'+tuple.data.RID
```

Comment:
Shouldn't we use `tuple.uniqueId` instead of  `tuple.data.RID` ? They mean the same thing for the tables that have `RID`. Besides I feel like we should avoid making assumption about name of columns in chaise.

Reply:
Also why not use the `tuple.displayname` if we want uniqueness? Unless we actually want to show the RID to the users, which then the filename format makes sense.

Reply:
We want an unformatted unique identifier for the row. We don't care what the identifier is as long as it's unique. So using `tuple.uniqueId` will be sufficient. We are using `table.name` because it doesn't include any special formatting from markdown or HTML.